README.md

```diff
@@ -1,4 +1,4 @@
-# ChatGPT + Enterprise data with Azure OpenAI and Cognitive Search
+# ChatGPT + Enterprise data with Azure OpenAI and Cognitive Search (.NET)
```

Comment:
Can we call it "ChatGPT"? I get why we'd do it (from an associated branding) but OpenAI has filed a trademark request for ChatGPT (which seems to have been denied and they are appealing), so it's possibly in a legal grey area at the moment.

Reply:
We're following similar naming as here but good question. 

https://github.com/Azure-Samples/azure-search-openai-demo

Tracking here #196